spec/cli_v1_commands.yaml

```
@@ -1,98 +1,555 @@
-#   info: general information about the library
-#   functions: public functions which the user can access
-#   types: a list of custom types
-
-clispec: '001'
-
-# This section outlines the general information for the library.
+clispec: "001"
 info:
   id: cli
-  version: 'next'
-  title: 'CLI'
-  language: 'sh'
-  source: 'https://github.com/supabase/cli'
-  bugs: 'https://github.com/supabase/cli/issues'
-  spec: 'https://github.com/supabase/supabase.tools/cli_spec/lib.yaml'
-  description: |
-    Supabase CLI.
-  options: | # Global options
-    ```sh
-    -v    --verbose               verbose output
-    ```
-
+  version: 1.0.0
+  title: Supabase CLI
+  language: sh
+  source: https://github.com/supabase/cli
+  bugs: https://github.com/supabase/cli/issues
+  spec: https://github.com/supabase/supabase.tools/cli_spec/lib.yaml
+  description: 'Supabase CLI '
+  options: |-
+    ```sh
+          --debug   output debug logs to stderr
+      -h, --help    help for supabase
+    ```
 commands:
-  - id: 'supabase-functions'
-    title: 'supabase functions'
-    summary: 'Manage your Edge Functions'
-    source: 'https://github.com/supabase/cli'
-    description: |
-      Manage your Edge Functions.
-    tags: []
-    links: []
-    usage: |
-      ```sh
-      supabase functions [command] [flags]
-      ```
-    subcommands:
-      - 'supabase-functions-new'
-      - 'supabase-functions-deploy'
-      - 'supabase-functions-delete'
-    options: |
-      ```
-      -h, --help   help for functions
-      ```
-
-  - id: 'supabase-functions-new'
-    title: 'supabase functions new'
-    summary: 'Create an Edge Functions'
-    source: 'https://github.com/supabase/cli'
-    description: |
-      Create an Edge Functions.
-    tags: []
-    links: []
-    usage: |
-      ```sh
-      supabase functions new <Function name>
-      ```
-    subcommands: []
-    options: |
-      ```
-      -h, --help   help for functions
-      ```
-
-  - id: 'supabase-functions-deploy'
-    title: 'supabase functions deploy'
-    summary: 'Deploy an Edge Functions'
-    source: 'https://github.com/supabase/cli'
-    description: |
-      Deploy an Edge Functions.
-    tags: []
-    links: []
-    usage: |
-      ```sh
-      supabase functions deploy <Function name>
-      ````
-    subcommands: []
-    options: |
-      ```
-      -h, --help   help for functions
-      ```
-
-  - id: 'supabase-functions-delete'
-    title: 'supabase functions delete'
-    summary: 'Delete an Edge Functions'
-    source: 'https://github.com/supabase/cli'
-    description: |
-      Delete an Edge Functions.
-    tags: []
-    links: []
-    usage: |
-      ```sh
-      supabase functions delete <Function name>
-      ```
-    subcommands: []
-    options: |
-      ```
+- id: supabase-stop
+  title: supabase stop
+  summary: Stop the Supabase local development setup
+  tags: []
+  links: []
+  usage: |-
+    ```sh
+    supabase stop [flags]
+    ```
+  subcommands: []
+  options: |-
```

```suggestion
  options: |
```

I don't think the `-` is required right? Not sure if it matters?

---

Yup `-` is optional. Adding a link for future reference https://yaml-multiline.info/